devtools/src/main/scala/kalix/devtools/impl/TracingConfExtractor.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.devtools.impl
+
+import com.typesafe.config.{ ConfigFactory, ConfigParseOptions, ConfigSyntax }
+
+import scala.util.matching.Regex
+
+object TracingConfExtractor {
+
+  val PortPattern: Regex = """(\d{1,5})""".r
```

Comment:
A better pattern would be, from 1 to 5 digits, occurring after a `:`.

Reply:
added